ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/FrontierGasCalculator.java

```diff
@@ -444,7 +445,8 @@ private Gas copyWordsToMemoryGasCost(
       final Gas wordGasCost,
       final UInt256 offset,
       final UInt256 length) {
-    final UInt256 numWords = length.dividedCeilBy(Bytes32.SIZE);
+    final int mod = length.mod(Bytes32.SIZE).isZero() ? 0 : 1;
```

Comment:
We do the divide ceil in multiple places.  If not in Tuweni perhaps we should have this in one static method as part of the superclass.

Reply:
Implemented now.